binr/radare2/radare2.c

```diff
@@ -83,7 +83,7 @@ static ut64 getBaddrFromDebugger(RCore *r, const char *file) {
 	if (!r || !r->io || !r->io->desc) {
 		return 0LL;
 	}
-#if __WINDOWS__
+#if 0
```

Comment:
if we can confirm the debugger works on windows with siol, this if0 block must be removed